modules/ocf_syslog/files/logrotate

```diff
@@ -1,9 +1,10 @@
-/var/log/remote/* {
-	daily
-	missingok
-	rotate 14
-	compress
+/var/log/remote/*.log {
+    daily
+    missingok
+    rotate 14
+    compress
     delaycompress
+    create 0640 root ocfstaff
```

Comment:
imo we probably don't want this, since it means any stream that ever existed will continue to exist forever (in empty rotated log files), even stuff like typo'd stream names (that's why i originally left it out)